### PR TITLE
Support NUMAKER_IOT_M487 V1.3 (re-raise)

### PR DESCRIPTION
**Pull request by ccli8**
## Description

This PR is re-raised to support NUMAKER_IOT_M487 V1.3.

**Note1**: #41 breaks for some unknown reason. This PR is re-raised to replace it.
**Note2**: Compared to V1.2, V1.3 adds ESP8266 RTS/CTS/RST wiring. It is incompatible with V1.2.

### CODE REVIEW

**Comment by iot-pdmc** [comment]
Thanks for your changes.
@ARMmbed/pdmc-integrators please review.

File: README.md [hunk 1]
```diff
@@ -26,6 +26,7 @@ Board                          |  Connectivity     | Storage for credentials and
 `Embedded Planet EP_AGORA`          | Cellular          | SPIF                    | Build-only
 `ST NUCLEO_H743ZI2`                 | Ethernet          | Internal Flash          | Build-only
 `ST NUCLEO_L4R5ZI`                  | Wi-Fi ESP8266     | Internal Flash          | Build-only
+`Nuvoton NUMAKER_IOT_M487`          | Wi-Fi ESP8266     | SD card (NUSD)          |
```
Comment (teetak01) [inline README.md]: This could have Build-only until we have CI-support in place.

**Comment by teetak01** [comment]
build-only

**Comment by iot-pdmc** [comment]
# [Test run: SUCCESS](https://jenkins-client.isgtesting.com/job/mbed-os-example-pelion/528/)
***
[Build artifacts](http://pdmc-ci.s3-website-eu-west-1.amazonaws.com/?prefix=jenkins-ci/mbed-os-example-pelion/528)

posted-by-tag: https://jenkins-client.isgtesting.com/job/mbed-os-example-pelion/

File: README.md (hunk 1 above)
Comment (ccli8) [reply README.md]: @teetak01 Changed to build-only

**Comment by teetak01** [comment]
build-only

File: README.md (hunk 1 above)
Comment (teetak01) [reply README.md]: Thanks

**Comment by iot-pdmc** [comment]
# [Test run: SUCCESS](https://jenkins-client.isgtesting.com/job/mbed-os-example-pelion/529/)
***
[Build artifacts](http://pdmc-ci.s3-website-eu-west-1.amazonaws.com/?prefix=jenkins-ci/mbed-os-example-pelion/529)

posted-by-tag: https://jenkins-client.isgtesting.com/job/mbed-os-example-pelion/

**Pull request merged**